doc/simastrom.tex

```diff
@@ -30,7 +30,7 @@
 \def\rm{\normalfont \rmfamily}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\title{Documentation of the meas/simastrom package}
+\title{Documentation of the jointcal package}
```

Comment:
I was thinking of a minor change of title to "Documentation of the astrometry component of the jointcal package".